tests/cluster/mount/__init__.py

```diff
@@ -1,6 +1,34 @@
-"""
-This folder does not contain any unittest code.
+from atc.config_master import TableConfigurator
+from atc.functions import init_dbutils
+from atc.spark import Spark
 
-The json file in this folder is passed as an argument to the setup job
-that configures the test environment. Therefore, the mount code is tested.
-"""
+
+def prepare_environment():
```

Comment:
We could consider making this as an actual function that others can use too?

make ```scope, storageAccount,...``` as input parameters.

Maybe also the *"abfss://silver@..."* - this could depend on the storage account setup.

In this way other projects could benefit from it - not only this atc deployment setup.

Reply:
The functions is extremely simple and taken directly from  the azure documentatiton here https://learn.microsoft.com/en-us/azure/databricks/data/data-sources/azure/azure-storage#--access-azure-data-lake-storage-gen2-or-blob-storage-using-oauth-20-with-an-azure-service-principal
I'm not sure it makes sense to hide this in an atc abstraction since the microsoft documentation is very good.